slack_bolt/adapter/aws_lambda/chalice_handler.py

```diff
@@ -15,6 +18,7 @@ def __init__(self, app: App, chalice: Chalice):
         self.app = app
         self.chalice = chalice
         self.logger = get_bolt_app_logger(app.name, ChaliceSlackRequestHandler)
+        self.app.lazy_listener_runner = ChaliceLazyListenerRunner(logger=self.logger)
```

Comment:
This code replaces the default (thread-based) one with Chalice (=Lambda framework by AWS) specific one.